core/src/test/java/ai/timefold/solver/core/api/solver/SolverManagerTest.java

```diff
@@ -515,13 +515,15 @@ void testScoreCalculationCountForFinishedJob() throws ExecutionException, Interr
 
         solverJob.getFinalBestSolution();
         assertThat(solverJob.getScoreCalculationCount()).isEqualTo(5L);
+        assertThat(solverJob.getMoveEvaluationCount()).isEqualTo(4L);
```

Comment:
That's because we calculate the score without applying any moves during the solve start phase:

1. ai.timefold.solver.core.impl.solver.DefaultSolver#solvingStarted
2. ai.timefold.solver.core.impl.solver.AbstractSolver#solvingStarted
3. ai.timefold.solver.core.impl.solver.recaller.BestSolutionRecaller#solvingStarted (line 50)

Reply:
A comment added to the test would help avoiding this question in the future.